aasemble/django/apps/buildsvc/models.py

```diff
@@ -328,6 +328,7 @@ def long_name(self):
     def name(self):
         return self.git_url.split('/')[-1].replace('_', '-')
 
+    @property
     def build(self):
```

Comment:
This should not be property. Calling .build() on a package source would effectively mean taking the return value of tasksbuild.delay(self.id) and trying to run that as a function with no arguments.

Making it a property means that the method would get called simply by invoking `.build` (rather than `.build()`). This is undesirable.